packages/aws-cdk/test/api/hotswap/state-machine-hotswap-deployments.test.ts

```diff
@@ -281,13 +281,14 @@ test('can correctly hotswap old style synth changes', async () => {
   });
 
   // WHEN
+  setup.pushStackResourceSummaries( setup.stackSummaryOf('Machine', 'AWS::StepFunctions::StateMachine', 'mock-state-machine-arn'));
```

Comment:
```suggestion
  setup.pushStackResourceSummaries(setup.stackSummaryOf('Machine', 'AWS::StepFunctions::StateMachine', 'mock-state-machine-arn'));
```

Reply:
Again, let's make these actual ARNs.

Reply:
And let's make them consistent with the physical name of the resource, not different.